diff --git a/utils/sascorer.py b/utils/sascorer.py
index 328e717..6ca1990 100644
--- a/utils/sascorer.py
+++ b/utils/sascorer.py
@@ -19,8 +19,7 @@ from __future__ import print_function
 
 from rdkit import Chem
 from rdkit.Chem import rdMolDescriptors
-from rdkit.six.moves import cPickle
-from rdkit.six import iteritems
+import pickle as cPickle
 
 import math
 from collections import defaultdict
@@ -60,7 +59,7 @@ def calculateScore(m):
   fps = fp.GetNonzeroElements()
   score1 = 0.
   nf = 0
-  for bitId, v in iteritems(fps):
+  for bitId, v in fps.items():
     nf += v
     sfp = bitId
     score1 += _fscores.get(sfp, -4) * v
@@ -177,4 +176,4 @@ def compute_sa_score(rdmol):
     sa = calculateScore(rdmol)
     sa_norm = round((10-sa)/9,2)
     return sa_norm
-    
\ No newline at end of file
+    
